package.json

```diff
@@ -2250,6 +2250,13 @@
 						"scope": "window",
 						"order": 26
 					},
+					"gitlens.graph.showPullRequests": {
```

Comment:
Lets call it `gitlens.graph.pullRequests.enabled` to align with the other PR related options

Reply:
Done in commit: [Rename setting](https://github.com/gitkraken/vscode-gitlens/pull/2450/commits/52d8b0ad6b5a0d33781410dae49af11c5fba1dc8)